string TEST_MICROPHONE = test-microphone
string NEW_DEMONSTRATION = new-demonstration
string TAKE_TOOL = take_tool
string DETECT_SURFACE = detect-surface
string START_RECORDING = start-recording
string STOP_RECORDING = stop-recording
string REPLAY_DEMONSTRATION = replay-demonstration
string RELEASE_TOOL = release-tool
string SAVE_ARM_POSE = save-arm-pose
string command
